saved-sites/saved-sites-impl/src/main/java/com/duckduckgo/savedsites/impl/SavedSitesRepository.kt

```diff
@@ -275,8 +275,9 @@ class RealSavedSitesRepository(
     override fun getBookmarkById(id: String): Bookmark? {
         val bookmark = savedSitesEntitiesDao.entityById(id)
         return if (bookmark != null) {
+            val isFavourite = getFavoriteById(id)
             savedSitesRelationsDao.relationByEntityId(bookmark.entityId)?.let {
-                bookmark.mapToBookmark(it.folderId)
+                bookmark.mapToBookmark(it.folderId, isFavourite != null)
```

Comment:
Should the tests be updated to check this?

Reply:
I need to do a big clean up. Will take care of that later